test/bigtest/tests/query-file-uploader-test.js

```diff
@@ -111,4 +114,48 @@ describe('queryFileUploader component', () => {
       expect(queryFileUploaderInteractor.callout.errorCalloutIsPresent).to.be.true;
     });
   });
+
+  describe('error modal showing with invalid type file', () => {
+    beforeEach(async () => {
+      await initiateFileUpload([new File([], 'file.pdf')]);
+    });
+
+    it('should render error modal', () => {
+      expect(fileExtensionModal.isPresent).to.be.true;
+    });
+
+    it('should display correct header', () => {
+      expect(fileExtensionModal.header.text).to.equal(translation['modal.fileExtensions.blocked.header']);
+    });
+
+    describe('cancel button clicked', () => {
+      beforeEach(async () => {
+        await fileExtensionModal.cancelButton.click();
+      });
+
+      it('should close modal', () => {
+        expect(fileExtensionModal.isPresent).to.be.false;
+      });
+    });
+
+    describe('"Choose other files" button clicked', () => {
```

Comment:
Let's not hardcode the naming of things as they can be changed in the future and tests might not be updated.